tox.ini

```diff
@@ -8,7 +8,11 @@ python = 3.11: py311-{lint, test}
 [testenv:py311-lint]
 description = Run static checkers.
 extras = lint
+allowlist_externals=pipenv
 commands =
+    # install dependencies
+    pipenv install --dev --deploy
```

Comment:
Not sure if it matters, but pipenv gives the following notice:
> Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.

If there's no more elegant way to do this, maybe we should use `PIPENV_VERBOSITY=-1` to explicitly acknowledge it.

Reply:
The problem here was that `tox` creates an own environment. I tried to adapt the config in order to make it work, but wirhout knowing mcuh about `tox`. Just one thing that comes to my mind: if we added `PIPENV_VERBOSITY=-1` to the `.env` file than the config would affect both the actual environment creation and the tox environment creation. Passing the environment variable on console in the `tox.ini` would be a different syntax on each OS (e.g. `$Env:PIPENV_VERBOSITY=-1` in powershell) wouldn't it?

Reply:
We can add the env var to the tox environment only:
```diff
diff --git a/tox.ini b/tox.ini
index 02d374f..eb20c98 100644
--- a/tox.ini
+++ b/tox.ini
@@ -9,6 +9,8 @@ python = 3.11: py311-{lint, test}
 description = Run static checkers.
 extras = lint
 allowlist_externals=pipenv
+; suppress warning about pipenv running in a venv
+set_env = PIPENV_VERBOSITY=-1
 commands =
     # install dependencies
     pipenv install --dev --deploy
@@ -33,6 +35,8 @@ commands =
 description = Run doc tests and unit tests.
 extras = test
 allowlist_externals=pipenv
+; suppress warning about pipenv running in a venv
+set_env = PIPENV_VERBOSITY=-1
 commands =
     # install dependencies
     pipenv install --dev --deploy
```

At least, pipenv is smart enough to detect the venv and not create another one. I think that's good enough for now, but let's hope they improve support for each other in the future.